docs/source/includes/gcloud_setup.rst

```diff
@@ -1 +1,24 @@
 Follow the Windows, Mac OS X or Linux instructions to install gcloud on your local machine: https://cloud.google.com/sdk/
+
```

Comment:
It would be great if we could directly in-line the instructions for installing gcloud, but I'd rather not have to keep up with changes to the setup instructions, which do change slightly over time. Sending users to the cloud/sdk site is fine IMO.